etc/ota-server/src/main.rs

```diff
@@ -0,0 +1,37 @@
+use axum::Router;
+use local_ip_address::local_ip;
+use std::net::SocketAddr;
+use tower_http::{services::ServeDir, trace::TraceLayer};
+use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
+
+const MICRO_RDK_OTA_BIN: &str = "micro-rdk-server-esp32-ota.bin";
+const TARGET_DIR: &str = "target/xtensa-esp32-espidf";
+
+#[tokio::main]
+async fn main() {
+    tracing_subscriber::registry()
+        .with(
+            tracing_subscriber::EnvFilter::try_from_default_env().unwrap_or_else(|_| {
+                format!("{}=debug,tower_http=debug", env!("CARGO_CRATE_NAME")).into()
+            }),
+        )
+        .with(tracing_subscriber::fmt::layer())
+        .init();
```

Comment:
it goes to stdout and is configured the same as normal logging (ex `RUST_LOG=debug cargo r`, `RUST_LOG make serve-dev-ota`). in it's current use, it's the same as the log crate.

This program is from axum's examples, and the `tracing` ecosystem is used as much as `tokio` is in rust web development. It can easily tie into language-agnostic log collectors, syslogs, etc. 

can port it to use just the log crate or something if you want

Reply:
No, if this is canonical for axum then that's fine.